es/index.json

```diff
@@ -286,7 +286,7 @@
   },
   "menu": {
     "mining_tips": "Consejos de minado",
-    "send_feedback": "Enviar sugerencias",
+    "report_issue": "Enviar sugerencias",
```

Comment:
Ugh.. why do we need to change our internal constants? The only thing we need to change is just the translations really

Reply:
Figured that it'll be confusing. Will revert it back. 😅